addons/forestry/CfgVehicles.hpp

```diff
@@ -0,0 +1,36 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class Forestry_Menu {
+                displayName = "Forestry";
+                condition = "true";
+                statement = "";
+                icon = QPATHTOF(resources\tree.paa);
+                exceptions[] = {
+                    "isNotSwimming",
+                    "isNotInside",
+                    "isNotSitting",
+                    "isNotOnLadder",
+                    "isNotRefueling"
+                };
+
+                class Chop_Tree {
+                    displayName = "Chop down tree";
```

Comment:
```suggestion
                    displayName = "Cut Down Tree";
```
Why is this different than keybind names.